db/models.py

```diff
@@ -46,3 +56,78 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE
+    )
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return str(self.created_at)[:19]
```

Comment:
I did it because of output example in README.md. There, we have output without milliseconds, and without slicing or strptime method I don't know how to achieve this? Or is this unnecessary?